### ISSUE TITLE
[EN][Item]Turtle Egg (Loggerhead)

### ISSUE THREAD

**Issue by Silvrette** [Atlantiss/BugTracker#9090]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**:  The Turtle Egg (Loggerhead) won't teach you the companion. 
http://www.wowhead.com/item=18964/turtle-egg-loggerhead

**Current behaviour**: The item is just grey and you can't use it.

**Expected behaviour**: The item's supposed to teach you the snapjaw companion.
http://www.wowhead.com/spell=23429/loggerhead-snapjaw

**Steps to reproduce the problem**:

1.  Get the item.
2.  Try to use the item.
3.  See that nothing's happening.

**Revision**: 5357

**Comment by 1Andas** [comment]
Yeah I can confirm this, item dropped several times during my runs in Azshara Crater.

**Comment by ashrasmun** [comment]
http://www.wowhead.com/item=18964/turtle-egg-loggerhead#comments
"Datamined from live realms. Not available to players."

**Comment by Piklo** [comment]
"Not available to players" not available because no longer can drop or something but it was one of azshara crater event loot so it should teach you a companion
